config/dependency-injection/services.php

```diff
@@ -28,6 +29,9 @@
 $container->register( WPSEO_Breadcrumbs::class, WPSEO_Breadcrumbs::class )->setAutowired( true )->setPublic( true );
 $container->register( WPSEO_Frontend::class, WPSEO_Frontend::class )->setAutowired( true )->setPublic( true );
 
+// Third-party classes
+$container->register( AccessTokenInterface::class, AccessTokenInterface::class )->setAutowired( true )->setPublic( true );
```

Comment:
Not sure why you're doing this, the `AccessTokenInterface` is an interface so can't be autowired and doesn't need to be registred.